slicebase/init/setup_rsync.sh

```diff
@@ -2,6 +2,20 @@
 
 SLICENAME=$1
 
+# A list of Google Cloud netblocks. Generated from DNS-based SPF records.  To
+# regenerate this list from DNS, you can run the command:
+#   nslookup -q=TXT _cloud-netblocks.googleusercontent.com  8.8.8.8 \
+#    | grep text \
+#    | sed -e 's/.*=spf1 //' -e 's/?all.*//' -e 's/include://g' -e 's/ /\n/g'  \
+#    | while read; do nslookup -q=TXT $REPLY 8.8.8.8; done \
+#    | grep 'text = ' \
+#    | sed -e 's/.*spf1 //' -e 's/ ?all.*//' -e 's/ /\n/g' \
+#    | grep ^ip4 \
+#    | sed -e 's/ip4://' \
+#    | xargs \
+#    | sed -e 's/ /, /g'
```

Comment:
I don't know how dynamic the list is. I suspect not very?

But I get a little twinge when I think of making rsync setup dependent on nslookup to 8.8.8.8 succeeding in just the right way. That's why I put the script in comments and the netblocks in code.